### DISCUSSION TITLE
Updating  DB connection information with API

### DISCUSSION THREAD

**Discussion by snvgglebear** [Budibase/budibase#6011] [Help]
the API documentation is not clear on what app parts can be updated with it. Can I update the DB connection information with the API?

**Comment by melohagan** [comment]
I think I answered this here: https://github.com/Budibase/budibase/discussions/6010#discussioncomment-2802770

But short answer no. Looking at the docs, it appears only the name and url of an app can be updated:
![Screenshot 2022-05-23 at 09 39 08](https://user-images.githubusercontent.com/101575380/169779308-ffe1b437-9985-4891-bf7f-26e2064ba150.png)

https://docs.budibase.com/reference/put_applications-appid
